recipes-kernel/linux/linux-raspberrypi.inc

```diff
@@ -102,51 +99,15 @@ do_configure_prepend() {
     CONF_SED_SCRIPT=""
 
     # oabi / eabi support
-    kernel_configure_variable AEABI y
     if [ "${ARM_KEEP_OABI}" = "1" ] ; then
         kernel_configure_variable OABI_COMPAT y
-    else
-        kernel_configure_variable OABI_COMPAT n
     fi
```

Comment:
We should document what kernels this change will work on. I am fine with change to limit itself to newer kernels.

Reply:
Not sure I understand your comment here. This is just removing the assignment of `OABI_COMPAT=n` when `ARM_KEEP_OABI` is not enabled since the default is already `n` in the kernel.

Long-term we should stop enabling this by default but I don't want to make such a potentially breaking change right now.

Reply:
I meant to say that if we are providing a knob to be able to use OABI that means we are supporting this ABI which seems to be not right. We should get rid of this setting completely, in long run we should be switch to yocto kernel tooling and drop the custom ways

Reply:
Fixed. I've added a final commit to drop OABI compatibility so it's easy to back out if that becomes necessary.